### PR TITLE
30 wakayama

### PR DESCRIPTION
**Pull request by shomiyazaki**
New SMC

Hi @kentoyamada100 ! This is the PR for the updated template with the new SMC.

Although the output of `> summary(sim_smc_pref_0)` returns an error, it is inevitable because of the geographical nature of this shapefile and the small n_dist. Even when I increase `n_sim` up to 100k, it still `shows` the same error.

`> summary(sim_smc_pref_0)
SMC: 10,000 sampled plans of 2 districts on 15 units
`adapt_k_thresh`=0.985 • `seq_alpha`=0.5
`est_label_mult`=1 • `pop_temper`=0.05

Plan diversity 80% range: 0.0 to 0.4
✖ WARNING: Low plan diversity

Sampling diagnostics for SMC run 1 of 2
         Eff. samples (%) Acc. rate Log wgt. sd  Max. unique Est. k 
Split 1     4,907 (98.1%)     32.3%        0.28 3,171 (100%)      8 
Resample    4,653 (93.1%)       NA%        0.28 3,081 ( 97%)     NA 

Sampling diagnostics for SMC run 2 of 2
         Eff. samples (%) Acc. rate Log wgt. sd  Max. unique Est. k 
Split 1     4,908 (98.2%)     32.2%        0.27 3,152 (100%)      8 
Resample    4,657 (93.1%)       NA%        0.27 3,087 ( 98%)     NA 

•  Watch out for low effective samples, very low acceptance rates (less than 1%),
large std. devs. of the log weights (more than 3 or so), and low numbers of unique
plans. R-hat values for summary statistics should be between 1 and 1.05.
• Low diversity: Check for potential bottlenecks. Increase the number of samples.
Examine the diversity plot with `hist(plans_diversity(sim_smc_pref_0), breaks=24)`.
Consider weakening or removing constraints, or increasing the population tolerance.
If the accpetance rate drops quickly in the final splits, try increasing `pop_temper`
by 0.01.`

### CODE REVIEW

**Comment by kentoyamada100** [comment]
Did you try loosening the population tolerance? Also, what did the histogram look like originally and what does it look like now?

**Comment by shomiyazaki** [comment]
@kentoyamada100 pop_tol is already very high (0.35) and the error won't disappear with 0.40.

Also, the histograms look like this.
Screen Shot 2022-06-05 at 18 09 10

File: analyses/30_wakayama/03_visualization_30_wakayama.R [hunk 1]
```diff
@@ -4,8 +4,9 @@
 ###############################################################################
```
Comment (kentoyamada100) [inline analyses/30_wakayama/03_visualization_30_wakayama.R:4]: `March 2022` -> `June 2022`

**Review by kentoyamada100 — CHANGES_REQUESTED** [review]
You don't need 30_wakayama_smc_10000_0.Rds, right? If so, pease delete i
Screen Shot 2022-06-12 at 15 09 38
t

**Comment by shomiyazaki** [comment]
@kentoyamada100 Thank you for your review. I have delegated the irrelevant codes

**Comment by shomiyazaki** [comment]
@kentoyamada100 Hi! Wakayama is ready with new `runs=4L`. I am looking forward to hearing your comments! Thank you!

**Review by kentoyamada100 — COMMENTED** [review]
Looks good. Please make sure to upload the new data to Dataverse. Also, please make a note somewhere that the plan diversity is low so that we can make sure to disclose that fact somewhere, when we make the results public

**Comment by shomiyazaki** [comment]
- [ ] Dataverse
- [ ] Plan Diversity

**Pull request merged**